src/haddock/modules/scoring/emscoring/__init__.py

```diff
@@ -56,6 +57,9 @@ def _run(self):
             expected_pdb = prepare_expected_pdb(
                 model, model_num, ".", "emscoring"
                 )
+            # fill the input.pdb - output.pdb dictionary
+            io_table[expected_pdb.file_name] = model.file_name
```

Comment:
The `PDBFile` class has `ori_name`, its best to use it instead:

```suggestion
            expected_pdb.ori_name = model.file_name
```

then you don't need the `io_table` dictionary

Reply:
perfect!